test/e2e/service_test.go

```diff
@@ -158,7 +158,7 @@ func (data *TestData) testNodePort(t *testing.T, isWindows bool, namespace strin
 	nodePort := int(svc.Spec.Ports[0].NodePort)
 	url := fmt.Sprintf("http://%s:%d", nodeIP, nodePort)
 
-	cmd := []string{"wget", "-O", "-", url, "-T", "1", "-t", "5"}
+	cmd := []string{"wget", "-O", "-", url, "-T", "3"}
```

Comment:
This method will still fail if it got `connrefused`.
Can we use code to retry explicitly or move to agnhost+curl?
You can refer to below code:
retry: https://github.com/gran-vmv/antrea/blob/ipam-vlan/test/e2e/service_test.go#L168
agnhost+curl: https://github.com/gran-vmv/antrea/blob/tf-win-e2e/test/e2e/traceflow_test.go#L1141

Reply:
I know, but I noticed that TestEgress is also like this. If I change the client to agnhost, then I need to make a lot of changes.

Reply:
Maybe we can just add a function `runWgetWithRetry` and call it at these places.

Reply:
@wenyingd Do we have `busybox` image for windows? If not, we should also consider move `busybox` to `agnhost` in future.

Reply:
e2eteam/busybox:1.29-windows-amd64-1809

If my remind is correnct, it is also ok that you use e2eteam/busybox:1.29 directly. docker would pull e2eteam/busybox:1.29-windows-amd64-1809

Reply:
Thank @wenyingd 
@hongliangl I think you can use `runWgetWithRetry` for this PR.
https://github.com/gran-vmv/antrea/blob/ipam-vlan/test/e2e/service_test.go#L168
And we should also add comments to inform future developers to check the busybox wget behavior on Windows.

Reply:
Thanks! @gran-vmv